src/interfaces/IResolver.sol

```diff
@@ -2,7 +2,7 @@
 
 pragma solidity ^0.8.0;
 
-import { Attestation } from "../Common.sol";
+import { Attestation, MultiAttestationRequest } from "../Common.sol";
```

Comment:
Where is the `MultiAttestationRequest` struct being used?

Reply:
Great catch bro. I have modified the multiAttest to not depends on it!